package com.stackroute.utility;

public class ExceptionCreation {
	public String firstException() {
		try {
			int a[]= new int[-20]; 
		}catch(NegativeArraySizeException error) {
			System.out.println(error.toString());
			return error.toString();
		}
		return null;
	}
	
	public String secondException() {
		try {
			int a[] = new int[5];
			a[6]=10;
		}catch(IndexOutOfBoundsException error) {
			System.out.println(error.toString());
			System.out.println(error.getClass().getName());
			return error.toString();
		}
		return null;
	}
	
	public String thirdException() {
		try {
			String str = null;
			str.length();
		}catch(NullPointerException error) {
			return error.toString();
		}
		return null;
	}

}
